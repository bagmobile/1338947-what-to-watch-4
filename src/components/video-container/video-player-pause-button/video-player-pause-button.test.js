import React from "react";
import renderer from "react-test-renderer";

import VideoPlayerPauseButton from "./video-player-pause-button";

describe(`VideoPlayerPauseButton component`, () => {

  it(`Render`, () => {
    const tree = renderer.create(
        <VideoPlayerPauseButton
          onClick={() => {
          }}
        />);
    expect(tree).toMatchSnapshot();
  });
});
