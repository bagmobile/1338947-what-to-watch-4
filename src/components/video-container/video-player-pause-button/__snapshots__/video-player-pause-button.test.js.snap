// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayerPauseButton component Render 1`] = `
<button
  className="player__play"
  onClick={[Function]}
  type="button"
>
  <svg
    height="21"
    viewBox="0 0 14 21"
    width="14"
  >
    <use
      xlinkHref="#pause"
    />
  </svg>
  <span>
    Pause
  </span>
</button>
`;
